cmd/servicelog/common.go

```diff
@@ -0,0 +1,55 @@
+package servicelog
+
+import (
+	"strings"
+
+	"github.com/openshift-online/ocm-cli/pkg/ocm"
+	sdk "github.com/openshift-online/ocm-sdk-go"
+	log "github.com/sirupsen/logrus"
+)
+
+var (
+	templateParams, userParameterNames, userParameterValues []string
+	isURL                                                   bool
+	HTMLBody                                                []byte
+)
+
+const (
+	targetAPIPath = "/api/service_logs/v1/cluster_logs" // https://api.openshift.com/?urls.primaryName=Service%20logs#/default/post_api_service_logs_v1_cluster_logs
+)
+
+func createConnection() *sdk.Connection {
+	connection, err := ocm.NewConnection().Build()
+	if err != nil {
+		if strings.Contains(err.Error(), "Not logged in, run the") {
+			log.Fatalf("Failed to create OCM connection: Authetication error, run the 'ocm login' command first.")
+		}
+		log.Fatalf("Failed to create OCM connection: %v", err)
+	}
+	return connection
+}
+
+func sendRequest(request *sdk.Request) *sdk.Response {
+	response, err := request.Send()
+	if err != nil {
+		log.Fatalf("Can't send request: %v", err)
+	}
+	return response
+}
+
+func check(response *sdk.Response, dir string) {
+	status := response.Status()
+
+	body := response.Bytes()
+
+	if status < 400 {
+		validateGoodResponse(body)
```

Comment:
@georgettica - I had a look at the 'why is the cluster list being printed out' problem - it's happening as a result of this call. This function always performs a dump of the response body contents to `stdout` ([link to the code](https://github.com/openshift/osdctl/blob/4e873844ea7b6bda85b15f76d16b79e56a37ca39/cmd/post.go#L308)).

The `validateGoodResponse()` function could maybe be refactored to instead pass `nil` to the `dump.Pretty()` function if no output is desired.. maybe via a `debug` control parameter or something.

Reply:
Thank you!
I will update it accordingly :))

Reply:
Was this updated? If so, make sure to resolve the comment

Reply:
I didn't do it and cleaned the use of `dump.Pretty()` so resolving and maybe in the future we can add more verbosity information